src/merge.js

```diff
@@ -0,0 +1,3 @@
+import merge from 'lodash.merge';
+
+export default merge;
```

Comment:
Mm, I wouldn't mind a babel plugin that injects `Object.assign` either. I guess the benefit of `Object.assign` is it's standard ES6 syntax so we won't have to worry about imports.

@alitaheri @oliviertassinari I agree with `assign` over `merge`, but do either of you feel strongly one way or the other lodash v object.assign w/ babel?